app/models/hunter.rb

```diff
@@ -19,7 +19,6 @@
 #  playbook_id :bigint
 #  user_id     :bigint
 #
-# The character appearing in the mystery
```

Comment:
That's a top-level comment that we want to keep.

Reply:
Resolved with commit 817d7cc. Tested with the annotate gem and top level documentation was not removed.